docs/sources/analyzing-profiles-in-pyroscope-ui/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "How to use the Pyroscope UI for performance analysis"
+menuTitle: "How to use the Pyroscope UI"
+description: "How to use the Pyroscope UI to analyze performance of your applications."
+weight: 30
+keywords:
+  - pyroscope
+  - UI
+  - performance analysis
+  - flamegraphs
+---
+
+# How to use the Pyroscope UI for performance analysis
+
+## Introduction
```

Comment:
Consider removing this heading and instead having a sentence or two introducing the benefit a read will gain from reading through this page and using this content. 

Also, what is the logical progression that someone needs to do to analyze app performance? (If that truly is the purpose  of this page. 

```suggestion
```